fractal_server/config.py

```diff
@@ -155,6 +155,7 @@ def DATABASE_SYNC_URL(self):
 
     RUNNER_CONFIG: str = "local"
     RUNNER_DEFAULT_EXECUTOR: str = "cpu-low"
+    RUNNER_PARSL_ENV_ACTIVATE: str = ""
```

Comment:
I'd rather use `sys.executable {path to process_worker_pool.py}` rather than forcing the fractal-server admin to explicitly set this variable.

Reply:
Fixed (including a warning about the scenario where users' jobs on computing nodes don't have access to the current `sys.executable`).